Sources/Certificate Pinning/TrustValidator.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  TrustValidator.swift
+//  Hyperspace-iOS
+//
+//  Created by Will McGinty on 1/30/19.
+//  Copyright © 2019 Bottle Rocket Studios. All rights reserved.
+//
+
+import Foundation
+
+@available(iOSApplicationExtension 10.0, *)
+@available(tvOSApplicationExtension 10.0, *)
+@available(watchOSApplicationExtension 3.0, *)
+
+/* This class performs validation on authentication challenges prestented to it. In addition to ensuring that the challenge is trusted by the operating system, it will
+    ensure that the certificate being presented as part of the SSL/TLS authentication challenge is recognized by the device. */
+public class TrustValidator {
+    
+    // MARK: - ValidationDecision Subtype
+    
+    public enum Decision {
+
```

Comment:
Should we remove this whitespace for consistency with how other enums are defined?